src/app/modules/registration/components/set-location-in-map/set-location-in-map.component.scss

```diff
@@ -57,6 +58,20 @@ app-map {
   gap: 10px;
 }
 
+/* Vi har ikke en bra støtte på veldig små skjermer, men dette burde holde bra nok for panoramisk viewet */
+@media screen and (max-height: 580px) and (min-width: 400px) {
```

Comment:
Virker som grensa hos meg for å få med alle knappene ligger på ca  `max-height: 650px` ?